src/platform/packages/shared/kbn-scout/src/playwright/runner/flags.ts

```diff
@@ -19,6 +19,7 @@ export interface RunTestsOptions {
   configPath: string;
   headed: boolean;
   mode: CliSupportedServerModes;
+  env: 'local' | 'cloud';
```

Comment:
Assuming developer may want to test against both ECH and MKI from local machine, it is easier to defined 2 json files with hosts/credentials instead of overriding one.
Then we can have something like `local | mki | ech` and expecting `.scout/servers` to have the same files e.g. `ech.json`.